oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -185,3 +185,38 @@ def change_power_state(uuid):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@computer_system.route(
+    "/redfish/v1/Systems/<uuid>", methods=["DELETE"])
+def remove_subscription(uuid):
+    """Removes a specific System
+
+        Args:
+            uuid: The System ID.
+
+        Exceptions:
+            HPOneViewException: When some OneView resource was not found.
+            return abort(404)
+
+            Exception: Unexpected error.
+            return abort(500)
+    """
+    try:
+        # Deletes server profile for given UUID
+        sucess = g.oneview_client.server_profiles.delete(uuid)
+
+        if not sucess:
+            abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+        return Response(
+            status=status.HTTP_200_OK,
```

Comment:
In this case, as there is no response body, we could return the status code 204 (No Content)

Reply:
Done